packages/functions/src/donor/CancelAppointmentHandler.ts

```diff
@@ -36,6 +36,7 @@ export default async function (
 
   appointment.donorId = "";
   const updatedAppointment = removeDonorFromDbAppointment(appointment);
+  updatedAppointment.status = AppointmentStatus.AVAILABLE;
```

Comment:
This change should be inside `removeDonorFromDbAppointment`.

This way it will also be used by `DeleteAppointmentsHandler` which is missing in this PR.

Also, please update the tests for `DeleteAppointmentsHandler` accordingly